### PR TITLE
Removing deprecated input parameter

### PR DESCRIPTION
**Pull request by simonwa7**

### CODE REVIEW

File: templates/psi4.yaml [hunk 1]
```diff
@@ -27,8 +27,6 @@ spec:
         default: 'False'
       - name: nthreads
         default: 1
-      - name: n-active
```
Comment (aakunitsa) [inline templates/psi4.yaml:30]: Could you please add `save-rdms` (default: False) to the template parameter list and to the function call (`save_rdms = {{inputs.parameters.save-rdms}}` )?

**Review by aakunitsa — CHANGES_REQUESTED** [review]
Except for a very minor change, it's good to go! Thanks for updating the template!

File: templates/psi4.yaml (hunk 1 above)
Comment (simonwa7) [reply templates/psi4.yaml:30]: Addressed here: c515960

Comment (aakunitsa) [reply templates/psi4.yaml:30]: Awesome, thanks!

**Review by aakunitsa — APPROVED** [review]

**Pull request merged**